turbine/src/broadcast_stage/broadcast_fake_shreds_run.rs

```diff
@@ -45,6 +45,21 @@ impl BroadcastRun for BroadcastFakeShredsRun {
             .expect("Database error")
             .map(|meta| meta.consumed)
             .unwrap_or(0) as u32;
+        let chained_merkle_root = match next_shred_index.checked_sub(1) {
+            None => broadcast_utils::get_chained_merkle_root(
+                bank.slot(),
+                bank.parent_slot(),
+                blockstore,
+            )
+            .unwrap(),
```

Comment:
We have some nice error handling in `get_chained_merkle_root` but attempt to just unwrap it here. Maybe add an expect or something? Is there any way to fail gracefully in the event we can't get the chained merkle root?

Reply:
> Maybe add an expect or something?

The `unwrap` will actually print the error. That is why I am adding those new error types, which are descriptive of what went wrong. So not `expect` will add much more here. This is code is also invoked only in tests.

> Is there any way to fail gracefully in the event we can't get the chained merkle root?

This code is only invoked in tests and it fails we want to get a loud panic so to debug root cause.